automatic/peazip.install/Readme.md

```diff
@@ -0,0 +1,7 @@
+﻿# <img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/b9b0ed4a/icons/peazip.svg" width="48" height="48"/> [PeaZip (Install)](https://chocolatey.org/packages/peazip)
+
+A cross-platform file archiver and file manager utility.
+
+The program supports over 150 archive formats including 7Z, ACE, ARC, ARJ, BZ2, CAB, DMG, GZ, ISO, LHA, PAQ, PEA, RAR, TAR, UDF, WIM, ZIP, ZIPX...
+
+Features of PeaZip includes extract, create and convert multiple archives at once, create self-extracting archives, split/join, encrypted password manager, strong encryption with two factor authentication, secure deletion, find duplicate files, calculate hashes, manage graphic files (rotate, crop, resize, convert).
```

Comment:
Features should be located under a `## Features` subheader.
See [1.2.3-informative-description][1.2.3]

[1.2.3]: https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description